common/test/package.scala

```diff
@@ -65,6 +66,7 @@ trait TestSettings {
 trait ConfiguredTestSuite extends ConfiguredServer with ConfiguredBrowser with ExecutionContexts {
   this: ConfiguredTestSuite with org.scalatest.Suite =>
 
+  lazy val webClient = new WebClient(BrowserVersion.CHROME)
```

Comment:
all we want is the html, seems very heavy duty to fire up a chrome just to get that.  we could use WS.url? or even just call the controller directly?